worlds/kh1/Client.py

```diff
@@ -0,0 +1,258 @@
+                    item_filename = f"AP_{str(item_num)}.item"
+                    for filename in os.listdir(self.game_communication_path):
+                        if filename == item_filename:
+                            found = True
+                    if not found:
+                        with open(os.path.join(self.game_communication_path, item_filename), 'w') as f:
+                            f.write(str(NetworkItem(*item).item) + "\n" + str(NetworkItem(*item).location) + "\n" + str(NetworkItem(*item).player))
+                            f.close()
+                            item_num = item_num + 1
+
+        if cmd in {"RoomUpdate"}:
+            if "checked_locations" in args:
+                for ss in self.checked_locations:
+                    filename = f"send{ss}"
+                    with open(os.path.join(self.game_communication_path, filename), 'w') as f:
+                        f.close()
+
+        if cmd in {"PrintJSON"} and "type" in args:
+            if args["type"] == "ItemSend":
+                item = args["item"]
+                networkItem = NetworkItem(*item)
+                recieverID = args["receiving"]
+                senderID = networkItem.player
+                locationID = networkItem.location
+                if recieverID != self.slot and senderID == self.slot:
+                    itemName = self.item_names[networkItem.item]
+                    itemCategory = networkItem.flags
+                    recieverName = self.player_names[recieverID]
+                    filename = "sent"
+                    with open(os.path.join(self.game_communication_path, filename), 'w') as f:
+                        f.write(
+                          re.sub('[^A-Za-z0-9 ]+', '',str(itemName))[:15] + "\n"
+                        + re.sub('[^A-Za-z0-9 ]+', '',str(recieverName))[:6] + "\n"
+                        + str(itemCategory) + "\n"
+                        + str(locationID))
+                        f.close()
+
+    def on_deathlink(self, data: dict[str, object]):
+        self.last_death_link = max(data["time"], self.last_death_link)
+        text = data.get("cause", "")
+        if text:
+            logger.info(f"DeathLink: {text}")
+        else:
+            logger.info(f"DeathLink: Received from {data['source']}")
+        with open(os.path.join(self.game_communication_path, 'dlreceive'), 'w') as f:
+            f.write(str(int(data["time"])))
+            f.close()
+
+    def run_gui(self):
+        """Import kivy UI system and start running it as self.ui_task."""
+        from kvui import GameManager
+
+        class KH1Manager(GameManager):
+            logging_pairs = [
+                ("Client", "Archipelago")
+            ]
+            base_title = "Archipelago KH1 Client"
+
+        self.ui = KH1Manager(self)
+        self.ui_task = asyncio.create_task(self.ui.async_run(), name="UI")
+
+
+async def game_watcher(ctx: KH1Context):
+    from worlds.kh1.Locations import lookup_id_to_name
```

Comment:
```suggestion
    from .Locations import lookup_id_to_name
```
Use relative imports within the world files for other world files